src/ErrorProne.NET.Core/ErrorProne.NET.Core.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Nullable>enable</Nullable>
+    <LangVersion>9.0</LangVersion>
```

Comment:
Usage of C# 9 breaks builds, because `global.json` restricts 3.1.200 SDK.

Tested on `mcr.microsoft.com/dotnet/core/sdk:3.1.200-alpine`

```
CSC : error CS1617: Invalid option '9.0' for /langversion. Use '/langversion:?' to list supported values. [/builds/marta/errorprone.net/ErrorProne.NET/src/ErrorProne.NET.Core/ErrorProne.NET.Core.csproj]
```